ci/Jenkinsfile

```diff
@@ -154,41 +154,41 @@ ansiColor('xterm') { withEnv(['TERM=xterm-color']) {
           }
         }
       },
-      macOS: {
-        setupStep('macos') { run ->
-          timeout(time: gotest_timeout, unit: 'MINUTES') {
-            run 'go get -v github.com/jstemmer/go-junit-report'
-            run "make gx-deps"
-
-            try {
-              run test + ' -tags="nofuse" 2>&1 | tee output'
-              run 'cat output | go-junit-report > junit-report-macos.xml'
-            } catch (err) {
-              throw err
-            } finally {
-              /* IGNORE TEST FAILS */
-              /* junit 'junit-report-*.xml' */
-            }
-          }
-        }
-      },
-      macSharness: {
-        setupStep('macos') { run ->
-          timeout(time: sharness_timeout, unit: 'MINUTES') {
-            run 'go get -v github.com/jstemmer/go-junit-report'
-            run "make gx-deps"
-
-            try {
-              run "make -j12 test/sharness/test-results/sharness.xml CONTINUE_ON_S_FAILURE=1 TEST_NO_FUSE=1"
-            } catch (err) {
-              throw err
```

Comment:
Seems just commenting this and line 167 would give the wanted effect, rather than commenting the entire thing. But since @magik6k initially developed it, I'll leave it up to him to decide.

Although, removing the builds fully feels wrong, as we want to make sure it at least runs somewhat on macOS

Reply:
It's better to comment out the whole blocks imo